src/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.Design/build/netstandard1.6/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.Design.targets

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
It's in the NuGet, but the file itself is never copied into a customer's project.